sys/posix/sockets/posix_sockets.c

```diff
@@ -238,6 +238,9 @@ static int _sockaddr_to_ep(const struct sockaddr *address, socklen_t address_len
             out->family = AF_INET6;
             memcpy(&out->addr.ipv6, &in6_addr->sin6_addr, sizeof(out->addr.ipv6));
             out->port = ntohs(in6_addr->sin6_port);
+            if(in6_addr->sin6_scope_id != 0) {
```

Comment:
While squashing please also note this style nit by vera++:

```
	sys/posix/sockets/posix_sockets.c:241: warning: keyword 'if' not followed by a single space
```

https://travis-ci.org/github/RIOT-OS/RIOT/builds/698438541#L191